tracking-exposed/yttrex
# fix(guardoni): experiment execution

To test this, please run:
```sh
yarn
rm -rf ~/.guardoni/config/profiles # remove previous profiles
yarn guardoni build
yarn guardoni dist:linux # dist:mac, dist:win
sudo dpkg -i ./platforms/guardoni/dist/guardoni_2.4.3_amd64.deb
yarn yt:backend watch
```
Open the "Guardoni" app and set "hadless" setting to `false`.

And then create an experiment by adding URL manually: it should create the experiment and run the browser.
